lib/graphql/language/generation.rb

```diff
@@ -116,7 +130,7 @@ def generate(node, indent: "")
         when Array
           "[#{node.map { |v| generate(v) }.join(", ")}]"
         when Hash
-          "{ #{node.map { |k, v| "#{k}: #{generate(v)}" }.join(", ")} }"
+          "{#{node.map { |k, v| "#{k}: #{generate(v)}" }.join(", ")}}"
```

Comment:
That's correct. I took for granted that Facebook's "kitchen sink" was formatted correctly. Also: https://github.com/graphql/graphql-js/blob/a725499b155285c2e33647a93393c82689b20b0f/src/language/printer.js#L81